src/controllers/auth.js

```diff
@@ -1,28 +1,37 @@
 const { tableService } = require('../services');
+const { rowService } = require('../services');
 const { dbTables } = require('../constants');
 
 const createDefaultTables = async () => {
-  //1. Check if the default tables are already created
+  // check if the default tables are already created
   const roleTable = tableService.checkTableExists('_roles');
   const usersTable = tableService.checkTableExists('_users');
   const rolesPermissionTable =
     tableService.checkTableExists('_roles_permissions');
   const usersRolesTable = tableService.checkTableExists('_users_roles');
 
-  //2. Create default auth tables
-  if (!roleTable) {
-    tableService.createTable('_roles', dbTables.roleSchema);
-  }
-
   if (!usersTable) {
+    // create the _users table
     tableService.createTable('_users', dbTables.userSchema);
   }
 
   if (!usersRolesTable) {
+    // create the _users_roles table
     tableService.createTable('_users_roles', dbTables.usersRoleSchema);
   }
 
-  if (!rolesPermissionTable) {
+  if (!roleTable && !rolesPermissionTable) {
```

Comment:
I think it's better if we separate these conditions just like the previous ones.